envoy/adapter/adapter.go

```diff
@@ -204,10 +204,15 @@ func envoyListenerFromService(svc *service.Service, envoyServiceName string,
 		return nil, fmt.Errorf("unrecognised proxy mode: %s", svc.ProxyMode)
 	}
 
-	serialisedConnectionManager, err := ptypes.MarshalAny(connectionManager)
+	// TODO: Switch to ptypes.MarshalAny when updating the Envoy API
+	serialisedConnectionManager, err := conversion.MessageToStruct(connectionManager)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create the connection manager: %s", err)
 	}
+	// serialisedConnectionManager, err := ptypes.MarshalAny(connectionManager)
+	// if err != nil {
+	// 	return nil, fmt.Errorf("failed to create the connection manager: %s", err)
+	// }
```

Comment:
Let's just remove this code and get it from the older commit when the time comes. You could add the commit reference to the comment.